src/forms/daoToDaoDelegateProposal.jsx

```diff
@@ -111,7 +100,7 @@ const DelegateProposalForm = ({
     const args = [
       UBERHAUS_DATA.ADDRESS,
       values.memberApplicant,
-      timePeriod?.toString() || values.delegateTerm,
+      '1705936544',
```

Comment:
could move this into a named constant, a little cryptic otherwise. JANUARY_2024 or something?